ramls/note.json

```diff
@@ -5,54 +5,54 @@
   "additionalProperties": false,
   "properties": {
     "id": {
-      "type": "string"
-    },
-    "link": {
       "type": "string",
-      "description": "Link to the item the note refers to",
-      "example": "/items/1234-567890"
+      "description": "Unique generated identified for the note",
```

Comment:
probably a typo in `identified` --> `identifier`

Reply:
Thanks Fixed